src/robot/parsing/model/statements.py

```diff
@@ -955,6 +955,40 @@ class End(NoArgumentHeader):
     type = Token.END
 
 
+@Statement.register
+class Continue(Statement):
+    type = Token.CONTIUNE
+
+    @classmethod
+    def from_params(cls, indent=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(Token.SEPARATOR, indent),
+            Token(Token.CONTIUNE),
+            Token(Token.EOL, eol)
+        ])
+
+    def validate(self):
+        if self.get_tokens(Token.ARGUMENT):
+            self.errors += ('CONTINUE does not accept arguments.',)
+
+
+@Statement.register
+class Break(Statement):
+    type = Token.BREAK
+
+    @classmethod
+    def from_params(cls, indent=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(Token.SEPARATOR, indent),
+            Token(Token.BREAK),
+            Token(Token.EOL, eol)
+        ])
+
+    def validate(self):
+        if self.get_tokens(Token.ARGUMENT):
+            self.errors += ('BREAK does not accept arguments.',)
```

Comment:
There's new `NoArgumentHeader` base class that both `Continue` and `Break` could inherit. It would provide both `from_params` and `validate` methods.

Reply:
Resolved. Modified code as per comment.